### DISCUSSION TITLE
Hide .dot files & directories

### DISCUSSION THREAD

**Discussion by vorburger** [PatrickF1/fzf.fish#112] [Archive] [answered]
It's showing .dot files & directories - I'd personally prefer it not to.. 

Presumably this is because you use `fd -H` - probably intentionally?

Perhaps https://github.com/PatrickF1/fzf.fish/wiki/FAQ could document how to make it hide .dot files & directories?

PS: Thanks for this GREAT tool!

**Comment by PatrickF1** [comment]
Hello. Glad you like it.
Yes I use `fd -H` intentionally because I wanted to set sane defaults for all users. However, I also didn't make it customizable, so there is currently no way to change that behavior unless you fork this repo and remove the flag. 😔
I am starting to change my mind about making it so un-configurable though. I may just add something that reads `FZF_FD_OPTS`. Sorry there's no clear solution right now.

Btw, one reason I'm hesitant to invest in the search file feature is because I want to keep it simple and not dip my toes any more than I have to into the file manager market. There are a ton of great file managers that will do better searching for you such as nnn. Just a recommendation.

**Comment by PatrickF1** [comment, marked as answer]
Hi! Guess what? I implemented this exact feature. See #126
